app/src/js/server/defaults.ts

```diff
@@ -9,13 +9,14 @@ export const serverConfig: ServerConfig = {
   database: 'local',
   userManagement: false,
   sync: false,
-  syncHost: 'http://localhost',
   autosave: true,
   redisTimeout: 3600,
   timeForWrite: 600,
   numActionsForWrite: 10,
   redisPort: 6379,
-  bots: false
+  bots: false,
+  pyHost: 'http://0.0.0.0',
```

Comment:
Just call it botHost?

Reply:
Could that cause confusion? Since the node js processes are called bots, I thought we should differentiate between them and the actual model server.